config/routes.rb

```diff
@@ -36,4 +36,15 @@
   get "/veterinary_offices/:veterinary_office_id/veterinarians", to: "veterinary_offices#veterinarians"
   get "/veterinary_offices/:veterinary_office_id/veterinarians/new", to: "veterinarians#new"
   post "/veterinary_offices/:veterinary_office_id/veterinarians", to: "veterinarians#create"
+
+  get "/applications/new", to: "applications#new"
+  post "/applications", to: "applications#create"
+  get "/applications/:id", to: "applications#show"
+  patch "/applications/:id", to: "applications#update"
+  
+  get "/:admin/applications/:id", to: "applications#show"
+  patch "/:admin/applications/:app_id/:pet_id/:app_rej", to: "application_pets#update"
+  
```

Comment:
a more RESTful route for this would be
```ruby
patch "/applications/:application_id/pets/:pet_id", to: "application_pets#update"
```
Let your view handle the if/else logic for `accept/reject`.